test_scripts/Polices/External_UCS_Informing_HMI/002_ATF_P_GetListofPermissions_with_appID_all_ecs_lower_bound_EntitySatus_ON.lua

```diff
@@ -50,9 +50,26 @@ end
 function Test:Precondition_PTU_and_OnAppPermissionConsent_AllParams_Lower()
   local ptu_file_path = "files/jsons/Policies/Related_HMI_API/"
   local ptu_file = "OnAppPermissionConsent_ptu.json"
```

Comment:
@Itileda, Thank you for clarification. Done in 410f7b5ffc29541a1c9dd8f4b372be304835855f